### ISSUE TITLE
cmyk().string() is different than expected

### ISSUE THREAD

**Issue by kaleeslivecn** [Qix-/color#201]
this will return rgb

**Comment by Qix-** [comment]
Yes because `color-string` doesn't yet support CSS color module level 4 yet, which is when `cmyk()` was introduced. Therefore the fallback is to return to `rgb`.

I suppose it's time to address this, but it has to be done in `color-string` first. I'll move this over there.
